src/cascadia/TerminalApp/AppKeyBindings.cpp

```diff
@@ -229,6 +235,8 @@ namespace winrt::TerminalApp::implementation
     DEFINE_EVENT(AppKeyBindings, SplitHorizontal,   _SplitHorizontalHandlers,   TerminalApp::SplitHorizontalEventArgs);
     DEFINE_EVENT(AppKeyBindings, IncreaseFontSize,  _IncreaseFontSizeHandlers,  TerminalApp::IncreaseFontSizeEventArgs);
     DEFINE_EVENT(AppKeyBindings, DecreaseFontSize,  _DecreaseFontSizeHandlers,  TerminalApp::DecreaseFontSizeEventArgs);
+    DEFINE_EVENT(AppKeyBindings, IncreaseZoom,      _IncreaseZoomHandlers,      TerminalApp::IncreaseZoomEventArgs);

```

Comment:
Shouldn't we just be using `IncreaseFontSize` and `DecreaseFontSize`? I'm pretty sure those events were put there for exactly this reason, despite not being hooked up.